doc/manuals/cameracalibration.rst

```diff
@@ -0,0 +1,83 @@
+.. _cameracalibration:
+
+=========================
+Manual Camera Calibration
+=========================
+
+This section describes the functions for manaul camera calibration capabilities in TeleSculptor (TS)
+and steps to achieve, possibly correct, and export a camera calibration model.
+
+Preparation
+===========
+
+TeleSculptor can work with ffmpeg-readable videos (e.g. mpg, mp4, avi) or with image sequences.
+The latter can be specified by listing paths to individual images in a plain text file (e.g. images.txt)
+using one line per path/to/image/file. Standard image formats (e.g. bmp, png, jpg) are supported.
+
+TeleSculptor has a capability to import PLY mesh or point cloud files via File/Import/Mesh.
+Point clouds are imported as faceless meshes.
+The imported mesh is used as a 3D target for calibration. RGB[A] color space is currently supported.
+The alpha channel can be used to set mesh transparency levels.
+
+We recommend geo-registering the mesh before importing it to TS to have the output calibrated
+camera parameters in mesh metric units and also automatically geo-registered with the 3D model.
+
+Calibration
+===========
+
+Start the TeleSculptor GUI and proceed with the following steps in a new or an existing project.
+
+.. |gcp_button| image:: /../gui/icons/22x22/location.png
+
+New project
+-----------
+
+Choose menu: File/New Project… (Ctrl+N) and select a folder for the new project,
+creating one if necessary. The project ini file name would have the same name by default.
+
+Choose menu: File/Import/Imagery… (Ctrl+O,I) and select a video or an image sequence text file.
+Large videos would imply a long search for metadata, which can be canceled via menu: Compute/Cancel.
+
+Choose menu: File/Import/Mesh… (Ctrl+O,M) and select an PLY file with a mesh or a point cloud,
+which should appear in the 3D viewer.
+
+Scroll to the desired video/image frame using the Camera Selection slider
+and observe it in the Camera View pane. Rotate the 3D model to the desired view,
+e.g. resembling the current image.
+
+To place/edit ground control points (GCP), click the 3D viewer GCP tool |gcp_button| 
+to start GCP creation and editing.
+Place six or more GCPs on the 3D model at the feature points visible from the camera view,
+observe their IDs appear in the Ground Control Points list, name them, if desired.
+
+To build 3D-2D point correspondences, click camera registration points (CRP) tool |gcp_button|
+in the Camera View.
+For each GCP in the list (also high-lighted in the 3D viewer):
+select a 3D point, then click on the Camera View image, where its 2D projection is expected to be.
```

Comment:
also mention Ctrl + click here to add CRPs

Reply:
mentioned Ctrl+click